01-gabbar/00-argocd-apps/01-dev/stakater-matomo.yaml

```diff
@@ -37,7 +37,7 @@ spec:
               enabled: 'false'
           auth:
             rootPassword: secretpassword
-  project: gabbar 
+  project: default 
```

Comment:
@faizanahmad055 why default project? here is should use gabbar

Reply:
@rasheedamir why was this change made? What is the correct choice to go with here?